platform/features/listview/src/directives/MCTGesture.js

```diff
@@ -0,0 +1,22 @@
+define(function () {
+    function MCTGesture(gestureService) {
+        return {
+            restrict : 'A',
+            scope: {
+                domainObject: '=mctObject'
+            },
+            link : function ($scope, $element, attrs) {
+                var activeGestures = gestureService.attachGestures(
+                    $element,
+                    $scope.domainObject,
+                    attrs.mctGesture.split(",")
```

Comment:
Having comma-separated values here makes the interface a little more complex/brittle than it needs to be. `"menu, info"` for instance would parse into something invalid (given that `" info"` is not a valid gesture key). It also, in effect, introduces new syntax for string arrays which is redundant to (albeit more concise than) the syntax already supported by JS/Angular.

[`$scope.$eval(attrs.mctGesture)`](https://docs.angularjs.org/api/ng/type/$rootScope.Scope#$eval) evaluates the value of `mct-gesture` as an Angular expression. Alternately, you couple include `mctGesture` in the `scope` block above, and [`$scope.$watch`](https://docs.angularjs.org/api/ng/type/$rootScope.Scope#$watch) for changes on it. 

`mct-gesture="menu,info"` becomes `mct-gesture="['menu', 'info']"`